content/release-10-14-0/platform-10-14-0-bundle/10-14-0-324.md

```diff
@@ -0,0 +1,35 @@
+---
+weight: 20
+title: Release 10.14.0.324
+layout: bundle
+---
+
+<!--10.14.0.316 - 10.14.0.324-->
+
+### Fixes
+
+<div><table ><colgroup>
+<col style="width: 15%;"><col style="width: 70%;"><col style="width: 15%;"></colgroup>
+<thead><tr>
+<th>
+Component</th>
+<th>
+Description</th>
+<th>
+Issue</th>
+</tr>
+</thead><tbody>
+
+<tr>
+<td>Authentication</td>
+<td>The performance of the first user requests sent after node restart has been improved. Prior to this change, there was an issue in rare cases where the number of all global roles in a tenant was close to but not exceeding 100, and many thousands of devices were concurrently trying to authorize MQTT connections on the restarted node.</td>
+<td>MTM-52049</td>
+</tr>
+
+<tr>
+<td>Core platform</td>
+<td>By adding or removing an external identifier of type <code>c8y_Serial</code> to/from the monitored device, the device connection monitoring is notified accordingly to start or stop with the specified external identifier.</td>

```

Comment:
this one is odd, had to read it several time to grasp what was meant.
"the monitored device" -> "a monitored device"
what does "start or stop with" mean?

Reply:
Same for me, I hoped that you get it better than I do :-). I have asked the developer in the ticket to rephrase the second half of the sentence.

Reply:
@rssag Better :-)?